instrumentation/net/http/httptrace/otelhttptrace/example/server/server.go

```diff
@@ -59,6 +59,7 @@ func main() {
 	helloHandler := func(w http.ResponseWriter, req *http.Request) {
 		ctx := req.Context()
 		span := trace.SpanFromContext(ctx)
+		defer span.End()
```

Comment:
This is the span extracted from the context for upstream. Since `defer span.End()` has already been executed at the underlying level, there is no need to perform it here.

FYI: https://github.com/open-telemetry/opentelemetry-go-contrib/blob/ad58c3a23580c5e441928602b45a447e692af481/instrumentation/net/http/otelhttp/handler.go#L129